src/modules/Backoffice/Toast.tsx

```diff
@@ -0,0 +1,56 @@
+import styled from 'styled-components';
+
+interface ToastProps {
+  message: string;
+  type: 'success' | 'error';
+}
+
+const ToastContainer = styled.div<{ type: 'success' | 'error' }>`
```

Comment:
I can see this being reusable in other places outside of Backoffice. Would it be possible to put it in components? Instead of being a module inside the Backoffice folder?